content/docs/building-apps/first-deposit.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Fund the Account and make the First Deposit
+order: 65
+---
+
+In this section, we will go over the new user account creation flow between non-custodial wallets and anchors with SEP-24 and/or SEP-6 implementations. Before we dive into the flow, it’s important to understand how Stellar accounts are created in the first place. 
+
+ The first step in creating a Stellar account is to   generate a keypair. This keypair includes public and private keys. However, an account does not exist and does not warrant space on the ledger until it’s  funded with the minimum balance of 1XLM by a Create Account operation. This requirement was created to minimize unused accounts from bloating the ledger. For a more in-depth look at this topic, check out this [page](https://developers.stellar.org/docs/tutorials/create-account/). 
```

Comment:
```suggestion
The first step in creating a Stellar account is to generate a keypair. This keypair includes public and private keys. However, an account does not exist and does not warrant space on the ledger until it’s funded with the minimum balance of 1XLM by a [Create Account](../../start/list-of-operations.mdx#create-acccount) operation. This requirement was created to minimize unused accounts from bloating the ledger. For a more in-depth look at this topic, check out this [page](../tutorials/create-account.mdx).
```

- nit: funky spacing, also best to keep our links relative (that way cross-references in the Preview Builder still work)
- add: link to operations whenever possible